starter_kits/Go/src/hlt/Game.go

```diff
@@ -11,13 +11,13 @@ import (
 type Game struct {
 	numPlayers int
```

Comment:
Any reason not to expose numPlayers too?

Reply:
Not that I can tell. It's easily accessible via len(Game.Players), but it's probably worth exposing directly for consistency if nothing else. I'll commit an update momentarily.

Reply:
Oh that's fair. Up to you anyway, I'm ok with either.

Reply:
I'll merge it as-is with the change in, but feel free to submit PR to revert it back to private.